tests/fixtures/typescript/basics/interface-with-all-property-types.src.ts

```diff
@@ -0,0 +1,15 @@
+interface Foo {
```

Comment:
A number of these aren't semantically valid TypeScript, but I guess we are further evaluating errorRecovery here, so it should be ok